src/modules/Remote.rb

```diff
@@ -235,25 +257,36 @@ def Write
     def configure_display_manager
       if IsEnabled()
         # Install required packages
-        if !Package.InstallAll(Packages.vnc_packages)
-          log.error "Installing of required packages failed"
-          return false
-        end
+        packages = Packages.vnc_packages
+        packages << PKG_CONTAINING_VNCMANAGER if Mode() == :vncmanager
 
-        # Enable xinetd
-        if !Service.Enable(XINETD_SERVICE)
-          Report.Error(
-            _("Enabling service %{service} has failed") % { service: XINETD_SERVICE }
-          )
+        if !Package.InstallAll(packages)
+          log.error "Installing of required packages failed"
           return false
         end
 
-        # Enable XDM
-        if !Service.Enable(XDM_SERVICE_NAME)
-          Report.Error(
-            _("Enabling service %{service} has failed") % { service: XDM_SERVICE_NAME }
-          )
-          return false
+        services = [
+          [XINETD_SERVICE, true],
+          [XDM_SERVICE_NAME, true],
+          [VNCMANAGER_SERVICE, @mode == :vncmanager]
```

Comment:
You sometime use ```Mode() == :vncmanager``` and sometime ```@mode == :vncmanager```. It is inconsistent and hard to maintain in case of further changes. Pls add ```Remote#vncmanager?``` & co (or something like that, naming methods is not my favorite thing to do).